src/org/usfirst/frc/team2335/robot/commands/AutoDriveAway.java

```diff
@@ -0,0 +1,51 @@
+package org.usfirst.frc.team2335.robot.commands;
+
+import org.usfirst.frc.team2335.robot.Robot;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.command.Command;
+
+public class AutoDriveAway extends Command 
+{
+	private double _distance; //Creates distance passing parameter which is the distance that the ultrasonic needs to be from the wall
+	//Defines are 
+    public AutoDriveAway(double distance)
+    {
+       requires(Robot.ultraSound);
+       requires(Robot.driveTrain);
+       
+       _distance = distance; 
+    }
+
+    // Called just before this Command runs the first time, We don't need to do anything
+    protected void initialize() 
+    {
+    	DriverStation.reportWarning("Started", true);
+    	DriverStation.reportWarning(Double.toString(_distance), true);
+    }
```

Comment:
Is reportWarning similar to SmartDashboard? How does it work and what will it display?